### ISSUE TITLE
Keyboard behavior on presenting view

### ISSUE THREAD

**Issue by hebertialmeida** [SwiftKickMobile/SwiftMessages#45]
Continuation of #36 

When the message comes-in, the keyboard disappear and when message comes-out, the keyboard appear again.

Would be really nice if SwiftMessages handle this, and keep the keyboard open and functional. May limit the UIWindow size for message.

**Comment by wtmoose** [comment]
Could you check out the head of master and let me know if this resolves the issue for you? I changed the default behavior of message windows to no longer become the key window. I added the following configuration option to `SwiftMessages.Config` in case anyone needs it to by the key window.
````swift
/**
 Specifies that in cases where the message is displayed in its own window,
 such as with `.window` presentation context, the window should become
 the key window. This option should only be used if the message view
 needs to receive non-touch events, such as keyboard input. From Apple's
 documentation https://developer.apple.com/reference/uikit/uiwindow:
 
 > Whereas touch events are delivered to the window where they occurred,
 > events that do not have a relevant coordinate value are delivered to
 > the key window. Only one window at a time can be the key window, and
 > you can use a window’s keyWindow property to determine its status.
 > Most of the time, your app’s main window is the key window, but UIKit
 > may designate a different window as needed.
 */
public var becomeKeyWindow = false
````

**Comment by hebertialmeida** [comment]
It is perfect now! Thank you so much....

Do you have idea when will you release these changes?

**Comment by wtmoose** [comment]
Cool. Will release in the next day or two. Waiting on another issue verification.

**Comment by hebertialmeida** [comment]
Alright.

**Comment by wtmoose** [comment]
This change is now in 3.1.2. Cheers
